os/drivers/wireless/realtek/wifi_util_interface/rtk_drv.c

```diff
@@ -174,9 +174,9 @@ int8_t wifi_scan_result_callback(wifi_utils_scan_list_s *utils_scan_input, int s
 	trwifi_scan_list_s *scan_list = (trwifi_scan_list_s *)utils_scan_input;
 
 	if (scan_list) {
-		lwnl_postmsg(LWNL_SCAN_DONE, (void *)scan_list);
+		TRWIFI_POST_SCANEVENT(LWNL_EVT_SCAN_DONE, (void *)scan_list);
```

Comment:
You used 'trwifi_post_event' for 'LWNL_EVT_SCAN_FAILED' and others.
But why did you use 'TRWIFI_POST_SCANEVENT' instead of 'trwifi_post_event' when event is 'LWNL_EVT_SCAN_DONE'
Is there any special reason?